egs/TAC/local/tac_dataset.py

```diff
@@ -0,0 +1,135 @@
+from torch.utils.data import Dataset
+import json
+import soundfile as sf
+import torch
+import numpy as np
+from pathlib import Path
+from asteroid.data.librimix_dataset import librispeech_license
+
+
+class TACDataset(Dataset):
+
+    dataset_name = "Multi-channel Librispeech-derived dataset used in Transform Average Concatenate"
```

Comment:
Maybe this description goes in the docstring, and we call it something smaller? 
The name will be used as a tag in the HF's hub.

Reply:
TACDataset ?